Sources/AppcuesKit/Push/PushMonitor.swift

```diff
@@ -94,7 +96,7 @@ internal class PushMonitor: PushMonitoring {
         #endif
 
         UNUserNotificationCenter.current().getNotificationSettings { [weak self] settings in
-            let shouldPublish = publishChange && self?.pushAuthorizationStatus != settings.authorizationStatus
+            let shouldPublish = self?.appcues?.sessionID != nil && self?.pushAuthorizationStatus != settings.authorizationStatus
```

Comment:
can you expand on this change? is that to ensure you are not starting a session on this event ?

Reply:
Basically yeah. There's [the code](https://github.com/appcues/appcues-ios-sdk/blob/5b3668a18154dcfc37a5b551658c29985a5f80e1/Sources/AppcuesKit/Data/Analytics/AnalyticsPublisher.swift#L55-L56) that starts a session if it can (ie there's a stored user id) and we don't want to do that in this case.

If there's a session and the value is different, that means that the DB has the old (now wrong) value and we want to update it.